src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -216,6 +216,18 @@ const useSubmitFormOnSsnChange = (
   ])
 }
 
+const getCheckoutErrorState = (error?: ApolloError) => {
+  if (!error) return null
+
+  const isManualReviewRequired = checkIsManualReviewRequired(
+    (error?.graphQLErrors || []) as GraphQLError[],
+  )
+
+  if (isManualReviewRequired) return 'MANUAL_REVIEW_REQUIRED'
```

Comment:
```suggestion
  if (checkIsManualReviewRequired(
    (error?.graphQLErrors || []) as GraphQLError[],
  )) {
      return 'MANUAL_REVIEW_REQUIRED'
  }
```

Reply:
Exactly same amount of information in less code.  I think the main point of having readable variables is documenting code

For example, this contains new info
```
const isManualReviewRequired = someUnclearCheck(...)
```

This, on the other hand, doesn't
```
const isManualReviewRequired = checkIsManualReviewRequired(...)
```

It's purely styling issue, so it's up to whatever you as an author like more